scripts/sfpshow

```diff
@@ -38,25 +40,6 @@ from utilities_common import multi_asic as multi_asic_util
 from utilities_common.platform_sfputil_helper import is_rj45_port, RJ45_PORT_TYPE
 
 # TODO: We should share these maps and the formatting functions between sfputil and sfpshow
```

Comment:
is this TODO is still valid? what is missing to close it?

Reply:
Yes, it is still valid. Now `show interface transceiver eeprom` and `sfputil show eeprom` have very similar output. However, these two commands have their own implementation. I suppose they shall share the same implementation, but I did not do it in this PR because it would be a big change.